src/syntax/parserEntry.ml

```diff
@@ -203,16 +209,14 @@ let parse_string com s p error inlined =
 			display_position := old_display;
 			in_display_file := old_in_display_file;
 		end;
-		Lexer.restore old;
-		display_error := old_de
+		Lexer.restore old
```

Comment:
Should we not save syntax_errors at beginning of parsing and restore them here as well ?

Reply:
Indeed, good catch